Gems/LyShine/Code/Source/UiSpawnerComponent.cpp

```diff
@@ -88,18 +90,20 @@ void UiSpawnerComponent::Reflect(AZ::ReflectContext* context)
         }
     }
 
-    AZ::BehaviorContext* behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context);
-    if (behaviorContext)
+    bool usePrefabSystem = false;
+    AzFramework::ApplicationRequests::Bus::BroadcastResult(usePrefabSystem, &AzFramework::ApplicationRequests::IsPrefabSystemEnabled);
+    if (!usePrefabSystem)
```

Comment:
Yes, since LyShine still uses the slice system, it still requires the UiSpawnerComponent and UiSpawner buses to allow spawning of UI slices. So we can leave this code as it was without the added "usePrefabSystem" check.